Doc/library/traceback.rst

```diff
@@ -653,5 +667,86 @@ This last example demonstrates the final few formatting functions:
    ['  File "spam.py", line 3, in <module>\n    spam.eggs()\n',
     '  File "eggs.py", line 42, in eggs\n    return "bacon"\n']
    >>> an_error = IndexError('tuple index out of range')
-   >>> traceback.format_exception_only(type(an_error), an_error)
+   >>> traceback.format_exception_only(an_error)
    ['IndexError: tuple index out of range\n']
+
+
+Examples of Using :class:`TracebackException`
+---------------------------------------------
+
+With the helper class, we have more options::
+
+   import sys
```

Comment:
Is this a REPL example or an example of how you might write a script? It appears to be somewhere between the two currently :-)

I'd make it a REPL example, similar to the two immediately above this section.